php/pantheon/checks/themes.php

```diff
@@ -54,42 +57,74 @@ public function run() {
 
 			$data = wp_get_theme($slug);
 			$version = $data->version;
-			$vulnerable = $this->is_vulnerable($slug, $version);
+			if ( $should_check_vulnerabilities ) {
+				$vulnerable = $this->is_vulnerable($slug, $version);
+			}
 
 			$needs_update = 0;
 			$available = '-';
 			if (isset($update[$theme_path])) {
 				$needs_update = 1;
 				$available = $update[$slug]->update["new_version"];
 			}
-			if ( false === $vulnerable ) {
-				$vulnerable = "None";
-			} else {
+			if ( $should_check_vulnerabilities && $vulnerable ) {
 				$vulnerable = sprintf('<a href="https://wpscan.com/themes/%s" target="_blank" >more info</a>', $slug );
+			} elseif ( $should_check_vulnerabilities ) {
+				$vulnerable = "None";
 			}
 
 			$report[$slug] = array(
 				'slug' => $slug,
 				'installed' => (string) $version,
 				'available' => (string) $available,
 				'needs_update' => (string) $needs_update,
-				'vulnerable'  => $vulnerable,
 			);
+
+			if ( $should_check_vulnerabilities ) {
+				$report[ $slug ]['vulnerable'] = $vulnerable;
+			}
```

Comment:
same as plugins above, this can be cleaned up to not check the same value three times.